libraries/AP_RangeFinder/AP_RangeFinder_BLPing.h

```diff
@@ -133,6 +133,13 @@ class AP_RangeFinder_BLPing : public AP_RangeFinder_Backend_Serial
      */
     void update(void) override;
 
+    /**
+     * @brief Get the reading confidence as a MAVLink "Quality" signal
+     * 0 is invalid, 100 is best quality, 1 is worst quality
```

Comment:
Let mavlink look after itself :-)  This method should be returning sensible thinks the mavlink library can interpret (which it does!)